crates/evm/src/tests/call_tests.rs

```diff
@@ -215,7 +221,7 @@ fn call_test() {
                 },
                 gas_used: 132943,
                 log_index_start: 0,
-                l1_diff_size: 565,
+                l1_diff_size: 1023,
```

Comment:
why only one of the transactions size changed?

Reply:
Fixed